src/components/MissingDocAlert.tsx

```diff
@@ -0,0 +1,127 @@
+                    padding="30px"
+                    border="2px"
+                    borderRadius="3px"
+                    borderColor={colourTheme.colors.Blue}
+                    backgroundColor="transparent"
+                    status="info"
+                >
+                    <InfoIcon height="70px" />
+                    <Box w="60%">
+                        {missingPOI && (
+                            <Box ml={4}>
+                                <Heading
+                                    mb={2}
+                                    size="md"
+                                    color={colourTheme.colors.Blue}
+                                >
+                                    {InfoCaption[0].heading}
+                                </Heading>
+                                <Text
+                                    display="block"
+                                    color={colourTheme.colors.Gray}
+                                >
+                                    {InfoCaption[0].desc}
+                                </Text>
+                            </Box>
+                        )}
+                        {missingCriminalCheck && (
+                            <Box ml={4}>
+                                <Heading
+                                    mb={2}
+                                    size="md"
+                                    color={colourTheme.colors.Blue}
+                                >
+                                    {InfoCaption[1].heading}
+                                </Heading>
+                                <Text
+                                    display="block"
+                                    color={colourTheme.colors.Gray}
+                                >
+                                    {InfoCaption[1].desc}
+                                </Text>
+                            </Box>
+                        )}
+                    </Box>
+                    <Box w="25%">
+                        <Flex direction="column" align="center">
+                            <Link href="/myaccounts">
+                                <Button
+                                    color="white"
+                                    backgroundColor={colourTheme.colors.Blue}
+                                    _hover={{
+                                        backgroundColor:
+                                            colourTheme.colors.LightBlue,
+                                    }}
+                                    size="sm"
+                                    py={5}
+                                    width="120%"
+                                    textDecoration="none"
+                                    borderRadius="6px"
+                                    fontWeight={"200"}
+                                >
+                                    View Details
+                                </Button>
+                            </Link>
```

Comment:
We use textDecoration: "none" to remove the underline 
and we want to wrap the link component with `next/link` for proper redirection.
```suggestion
                            <Link href="/myaccounts">
                                <ChakraLink _hover={{ textDecoration: "none" }}>
                                    <Button
                                        color="white"
                                        backgroundColor={
                                            colourTheme.colors.Blue
                                        }
                                        _hover={{
                                            backgroundColor:
                                                colourTheme.colors.LightBlue,
                                        }}
                                        size="sm"
                                        py={5}
                                        width="120%"
                                        textDecoration="none"
                                        borderRadius="6px"
                                        fontWeight={"200"}
                                    >
                                        View Details
                                    </Button>
                                </ChakraLink>
                            </Link>
```